common/services/prismic/exhibitions.js

```diff
@@ -262,6 +271,42 @@ export async function getExhibition(req: Request, id: string): Promise<?UiExhibi
   }
 }
 
+// TODO better naming
```

Comment:
Is there something this is doing, that this doesn't?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/multi-content.js